client/src/components/dashboard/verify/Verify.tsx

```
@@ -98,59 +125,47 @@
               <p className="mt-2 text-sm text-green-600">
-                File selected: {image.name}
+                Selected: {image.name}
               </p>
             )}
+
             <button
-              onClick={handleUpload}
-              className="mt-4 px-4 py-2 bg-blue-600 text-white rounded hover:bg-blue-800"
+              onClick={handleUpload} // Ensure userId is not null
+              className="mt-3 px-4 py-2 bg-blue-600 text-white rounded hover:bg-blue-800"
             >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)